packages/react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -456,6 +456,9 @@ class ContextualMenuInternal extends React.Component<IContextualMenuInternalProp
                 onKeyDown={this._onMenuKeyDown}
                 onKeyUp={this._onKeyUp}
                 onFocusCapture={onMenuFocusCapture}
+                aria-label={ariaLabel}
+                aria-labelledby={labelElementId}
+                role={'menu'}
```

Comment:
Shouldn't this be `role={role ?? 'menu'}` to keep in line with how it worked before?

Reply:
That's what I was thinking at first as well but `_onRenderMenuList` is never actually passed a `role` parameter and `ContextualMenu` doesn't take a `role` prop either so `role` was going to be `menu` every time anyway which is why i decided to remove the conditional 😅